doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -122,6 +122,46 @@ The following rules govern create, read, update, and delete (CRUD) level access.
 
 <a name="6.3.2"></a>
 ### 6.3.2 System Hardening
+All infrastructure should undergo system hardening, establishes processes to govern the hardening, and documents to cover at a minimal for the following areas:
+
+#### 6.3.2.1 Function
+Infrastructure should be implemented to perform the minimal function that’s practically needed to support NFVI. 
+
+#### 6.3.2.2 Software
+- Install only software which is required to support the functions
+- Remove any unnecessary software or packages
+- Where software cannot be removed, disable all service to it
+
+#### 6.3.2.3 Patches
```

Comment:
Stating these timelines is reasonable but it is really down to the Operator to decide.   Perhaps we should say 'suggested' resolution times.  Also is there a definition of each criticality level that we need to define or is it coming from security industry standards?

Reply:
The language is "should" instead of "must" to make it a suggestion.  I also included a note to the NVD's Common Vulnerability Scoring System now, thanks!